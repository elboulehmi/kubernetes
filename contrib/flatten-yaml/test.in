key1: value1
key2:
  key3: 
    key4: 
      key5: value5
key6:
  key7:
    - key8: value8
      key9: value9
    - key8: value8b
      key9: value9b
  key10: value10
