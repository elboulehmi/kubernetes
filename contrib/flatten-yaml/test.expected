.key1: value1
.key2.key3.key4.key5: value5
.key6.key7.0.key8: value8
.key6.key7.0.key9: value9
.key6.key7.1.key8: value8b
.key6.key7.1.key9: value9b
.key6.key10: value10
